ipmi/Sensor_Discovery/Template_IPMI_Sensor_Discovery.dummy.xml

```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<zabbix_export>
+    <version>3.0</version>
+    <date>2017-09-29T10:36:16Z</date>
+    <groups>
+        <group>
+            <name>IPMI templates</name>
+        </group>
+    </groups>
+    <templates>
+        <template>
+            <template>Template IPMI Sensor Discovery</template>
+            <name>Template IPMI Sensor Discovery</name>
+            <description>This is just an empty IPMI Sensor discovery dummy template, required for documentation generation. There's no need to install it.</description>
```

Comment:
I feel like a dummy template is a way to hacky solution for this. Why didn't you change the generator script to do it's thing without needing to render this dummy template?

Reply:
The problem is, that the current `xsltproc` based documentation generation needs an input XML to do it's work. Currently, `IPMI Sensor Discovery` is the only package without a corresponding Template XML and we probably won't have significantly more of them in the future.
However, if you have an alternative proposal which doesn't requires too much adaptions of the generation process, I will be more than happy to get rid of the dummy template.

Reply:
Thanks for the proposal, addressed with 41f62301e328242e9dbdcd0ed07664277dcf0496 and 9f5823da07cc3bee23f0c3b9fe7e30eac59123d4.